src/components/navigation/Navbar.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import CustomImage from "../utils/CustomImage";
+import Button from "../utils/Button";
+import NavLink from "./NavLink";
+import logo from "../../images/logo.svg";
+import hamburger from "../../images/icon-hamburger.svg";
+import NavDrawer from "./NavDrawer";
+import breakpoints from "../../utils/breakpoints";
+import { respondTo } from "../../utils/Mixins";
+
+const NavbarNav = styled.nav`
+  display: flex;
+  align-items: center;
+  margin: 2.125em 3em 0 2.5em;
+  color: #fff;
+  justify-content: space-between;
+
+  ${respondTo(
+    breakpoints.device.xs,
+    `
+    margin-left:  1.4375em;
+    margin-right:  1.4375em;
+  `
+  )}
+`;
+
+const NavLinks = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: nowrap;
+
+  @media screen and (max-width: 768px) {

```

Comment:
You should aim to use the same approach for media queried everywhere if it makes sense